packages/editor/src/components/post-publish-button/index.js

```diff
@@ -157,9 +158,15 @@ export class PostPublishButton extends Component {
 		}
 
 		const onClickButton = () => {
+			// Allow for overriding saving for custom validations.
+			if (!applyFilters('editor.PostPublishButton.shouldSubmit',true,this.props)) {
```

Comment:
The static analysis CI has me a little confused as to what change needs to be made here but I'm happy to make that change to get it passing.

Reply:
Yeah I'm not sure here -- I've adjusted it a few times but taking out spaces doesn't seem to get it passing and the https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/ code standards themselves aren't passing so I'm 🤷